integration-cli/docker_cli_by_digest_test.go

```diff
@@ -616,6 +617,8 @@ func (s *DockerSchema1RegistrySuite) TestPullFailsWithAlteredManifest(c *testing
 // This is the schema2 version of the test.
 func (s *DockerRegistrySuite) TestPullFailsWithAlteredLayer(c *testing.T) {
 	testRequires(c, DaemonIsLinux)
+	skip.If(c, testEnv.UsingSnapshotter(), "Faked layer is already in the content store, so it won't be fetched from the repository at all.")
```

Comment:
I'm good with skipping for now; would there still be a use to detect corrupted / manipulated data in the content-store (does containerd have anything for that?)